community/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/ProcedureCallSupportAcceptanceTest.scala

```diff
@@ -22,10 +22,81 @@ package org.neo4j.internal.cypher.acceptance
 import java.util
 
 import org.neo4j.cypher._
-import org.neo4j.kernel.api.proc.Neo4jTypes
 
 class ProcedureCallSupportAcceptanceTest extends ProcedureCallAcceptanceTest {
 
+  test("should work inside FOREACH") {
```

Comment:
We should add tests for:
 * `CALL` inside `FOREACH` inside `FOREACH`
 * `CALL ... YIELD` inside `FOREACH`
 * A `CALL` that takes input params from the variable introduced in the `FOREACH`
 * `NULL` is another thing that usually exposes problems, but I don't know how to get that in here